source/module_ri/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ if (ENABLE_LIBRI)
         exx_abfs-construct_orbs.cpp
         exx_abfs-io.cpp
         exx_abfs-jle.cpp
-        exx_abfs-pca.cpp
+        ABFs_Construct-PCA.cpp
```

Comment:
Should Makefile.Objects be modified at the same time?

Reply:
Yes, I missed this one.